recipes/gtk4-layer-shell/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -ex
+
+export LD_LIBRARY_PATH=$PREFIX/lib
```

Comment:
I would not expect this to be necessary. What happens if you do not specify this?